storage/elasticsearch/elasticsearch.go

```diff
@@ -40,17 +40,20 @@ type elasticStorage struct {
 }
 
 type detailSpec struct {
-	Timestamp      int64                `json:"timestamp"`
-	MachineName    string               `json:"machine_name,omitempty"`
-	ContainerName  string               `json:"container_Name,omitempty"`
-	ContainerStats *info.ContainerStats `json:"container_stats,omitempty"`
+	Timestamp       int64                `json:"timestamp"`
+	MachineName     string               `json:"machine_name,omitempty"`
+	ContainerName   string               `json:"container_Name,omitempty"`
```

Comment:
i know its not new to this pr, but its ashame we are not consistent with camel casing.